rdkit/Chem/AllChem.py

```diff
@@ -11,51 +11,54 @@
 """ Import all RDKit chemistry modules
 
 """
-from rdkit import rdBase
-from rdkit import RDConfig
+import warnings
+
+import numpy
+
 from rdkit import DataStructs
-from rdkit.Geometry import rdGeometry
+from rdkit import ForceField
+from rdkit import RDConfig
+from rdkit import rdBase
 from rdkit.Chem import *
-from rdkit.Chem.rdPartialCharges import *
-from rdkit.Chem.rdDepictor import *
-from rdkit.Chem.rdForceFieldHelpers import *
 from rdkit.Chem.ChemicalFeatures import *
+from rdkit.Chem.rdChemReactions import *
+from rdkit.Chem.rdDepictor import *
 from rdkit.Chem.rdDistGeom import *
+from rdkit.Chem.rdForceFieldHelpers import *
 from rdkit.Chem.rdMolAlign import *
+from rdkit.Chem.rdMolDescriptors import *
 from rdkit.Chem.rdMolTransforms import *
-from rdkit.Chem.rdShapeHelpers import *
-from rdkit.Chem.rdChemReactions import *
+from rdkit.Chem.rdPartialCharges import *
 from rdkit.Chem.rdReducedGraphs import *
+from rdkit.Chem.rdShapeHelpers import *
+from rdkit.Chem.rdqueries import *
+from rdkit.Geometry import rdGeometry
+from rdkit.RDLogger import logger
+
 try:
   from rdkit.Chem.rdSLNParse import *
 except ImportError:
   pass
-from rdkit.Chem.rdMolDescriptors import *
-from rdkit.Chem.rdqueries import *
-from rdkit import ForceField
+
 Mol.Compute2DCoords = Compute2DCoords
 Mol.ComputeGasteigerCharges = ComputeGasteigerCharges
-import numpy, os
-from rdkit.RDLogger import logger
 logger = logger()
-import warnings
 
 
 def TransformMol(mol, tform, confId=-1, keepConfs=False):
   """  Applies the transformation (usually a 4x4 double matrix) to a molecule
   if keepConfs is False then all but that conformer are removed
-
   """
   refConf = mol.GetConformer(confId)
   TransformConformer(refConf, tform)
   if not keepConfs:
     if confId == -1:
       confId = 0
     allConfIds = [c.GetId() for c in mol.GetConformers()]
-    for id in allConfIds:
-      if not id == confId:
-        mol.RemoveConformer(id)
-    #reset the conf Id to zero since there is only one conformer left
+    for id_ in allConfIds:
```

Comment:
Ah, confId is already used.  Perhaps cid or something else.  Honestly, id as a builtin I think is a bad feature in python.  It is too often used in practice.